### PR TITLE
fix: add test case to default html node

### PR DESCRIPTION
**Pull request by qinluhe**

### CODE REVIEW

File: collab-document/src/importer/md_importer.rs [hunk 1]
```diff
@@ -182,6 +182,11 @@ fn process_mdast_node(
     mdast::Node::Image(image) => process_image(document_data, image, &id),
     _ => {
       trace!("Unhandled node: {:?}", node);
```
Comment (LucasXu0) [inline collab-document/src/importer/md_importer.rs:184]: Should we keep this log as the node has been parsed to the default text paragraph?

File: collab-document/tests/importer/md_importer_test.rs [hunk 2]
```diff
@@ -453,3 +453,33 @@ fn test_table() {
     }
   }
 }
+
+#[test]
+fn test_aside() {
+  let markdown = r#"<aside>
```
Comment (LucasXu0) [inline collab-document/tests/importer/md_importer_test.rs:459]: It's the HTML tag, right? I tried to find if there's `parse_options` in `mdast` to enable parsing the callout, unfortunately, found nothing.

Comment (qinluhe) [reply collab-document/tests/importer/md_importer_test.rs:459]: yes.

File: collab-document/src/importer/md_importer.rs (hunk 1 above)
Comment (qinluhe) [reply collab-document/src/importer/md_importer.rs:184]: I think it's ok

**Pull request merged**